tests/test_fms_api_team_avatar_parser.py

```diff
@@ -0,0 +1,49 @@
+import json
+import unittest2
+
+from datafeeds.parsers.fms_api.fms_api_team_avatar_parser import FMSAPITeamAvatarParser
+
+from google.appengine.ext import ndb
+from google.appengine.ext import testbed
+
+from models.district import District
+
+
+class TestFMSAPITeamAvatarParser(unittest2.TestCase):
```

Comment:
Add another test that parses a whole event.

Reply:
Oh, good idea.  👌